Source/DafnyDriver/DafnyDriver.cs

```diff
@@ -412,6 +412,10 @@ public enum CommandLineArgumentsResult {
         if (stats.OutOfResourceCount != 0) {
           Console.Write(", {0} out of resource", stats.OutOfResourceCount);
         }
+        if (stats.SolverExceptionCount != 0) {
+          Console.Write(", {0} solver exceptions", stats.SolverExceptionCount);
```

Comment:
I see you're also handling the side effects of https://github.com/boogie-org/boogie/pull/488 - for the record I assume you weren't forced to by any build errors when you upgraded the version of Boogie, but just knew it was a good idea to sneak this in as well? :)

Reply:
Yeah, I made the change to Boogie to include that metric, and added a commit to this branch to account for it when I did.